test/presubmit-tests.sh

```diff
@@ -85,10 +85,23 @@ api-unittest(){
   info Running unittests
 
   go mod vendor
-  go test -p 1 -v ./pkg/... && 
+  go test -p 1 -v ./pkg/... &&
   go test -p 1 -v ./v1/service/...
 }
 
+api-golangci-lint() {
+  info Running go lint
+
+  go mod vendor
+  golangci-lint run ./pkg/... ./v1/service/...
+}
+
+yaml-lint() {
+  info Running Yamllint
+
+  yamllint -c .yamllint ./config.yaml ./api ./config
```

Comment:
running on `./api` necessary?

Reply:
Yes we do have manifest for fixtures, don't we need to apply lint check on that ?